src/Native/CMakeLists.txt

```diff
@@ -231,7 +241,11 @@ if(WIN32)
 endif()
 add_subdirectory(Runtime)
 add_subdirectory(Bootstrap)
-add_subdirectory(jitinterface)
+
+# We don't need jitinterface for WASM.
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Emscripten)
```

Comment:
Should we rather use `CLR_CMAKE_PLATFORM_ARCH_WASM` for these checks? (It would be self-explanatory at least - no need to have `We don't need jitinterface for WASM.` comment)